### PR TITLE
gh-85935: Explicitly document the case nargs=0 in argparse

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
* Issue: gh-85935

----
📚 Documentation preview 📚: https://cpython-previews--125302.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/argparse.rst [hunk 1]
```diff
@@ -731,6 +731,9 @@ how the command-line arguments should be handled. The supplied actions are:
 
   .. versionadded:: 3.8
 
+Only actions that consume command-line arguments (such as ``'store'``,
```
Comment (savannahostrowski) [inline Doc/library/argparse.rst]:
```suggestion
Only actions that consume command-line arguments (e.g. as ``'store'``,
```

File: Doc/library/argparse.rst [hunk 2]
```diff
@@ -858,6 +861,8 @@ See also :ref:`specifying-ambiguous-arguments`. The supported values are:
 If the ``nargs`` keyword argument is not provided, the number of arguments consumed
 is determined by the action_.  Generally this means a single command-line argument
 will be consumed and a single item (not a list) will be produced.
+Actions that do not consume command-line arguments (such as
```
Comment (savannahostrowski) [inline Doc/library/argparse.rst]:
```suggestion
Actions that do not consume command-line arguments (e.g.
```

**Review by savannahostrowski — CHANGES_REQUESTED** [review]
Two minor comments for conciseness, but otherwise, LGTM.

File: Doc/library/argparse.rst (hunk 1 above)
Comment (serhiy-storchaka) [reply Doc/library/argparse.rst]: I think "as" was redundant.
```suggestion
Only actions that consume command-line arguments (e.g. ``'store'``,
```

**Pull request merged**